stores/sql/metrics.go

```diff
@@ -25,13 +25,36 @@ func ContractPruneMetrics(ctx context.Context, tx sql.Tx, start time.Time, n uin
 		if err := rows.Scan(&placeHolder, &placeHolderTime, &timestamp, (*FileContractID)(&cpm.ContractID), (*PublicKey)(&cpm.HostKey), &cpm.HostVersion, (*Unsigned64)(&cpm.Pruned), (*Unsigned64)(&cpm.Remaining), &cpm.Duration); err != nil {
 			return nil, fmt.Errorf("failed to scan contract prune metric: %w", err)
 		}
-		cpm.Timestamp = api.TimeRFC3339(timestamp)
+		cpm.Timestamp = api.TimeRFC3339(normaliseTimestamp(start, interval, timestamp))
```

Comment:
Want to draw attention to this, wasn't 100% sure, this was an oversight right? We want to normalise all metric timestamps?